src/SavedSearch.php

```diff
@@ -302,7 +302,7 @@ public function rawSearchOptions()
      */
     public function prepareSearchUrlForDB(array $input): array
     {
-        $taburl = parse_url(rawurldecode(Sanitizer::unsanitize($input['url'])));
+        $taburl = parse_url(Sanitizer::unsanitize(rawurldecode($input['url'])));
```

Comment:
Where does the corresponding `rawurlencode` is done ? Maybe the best way to fix this is to `rawurlencode` something that is not already sanitized.

Reply:
at this place, i receive url with %XX (so url encoded) and url decoding is required
Not sure about unsanitize but it may depends on the searched content i think